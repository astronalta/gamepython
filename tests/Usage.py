import os

os.system(os.path.join("..", "bin", "jogo") + " --help")
os.system(os.path.join("..", "bin", "jogo") + " -h")
