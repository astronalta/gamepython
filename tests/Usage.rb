system("../bin/apollo --help")
system("../bin/apollo -h")
