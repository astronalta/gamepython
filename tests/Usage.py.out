Usage: jogo [OPTIONS] FILE...

   -a, --assembly       Compile, but do not assemble or link.
   -c, --compile        Compile and assemble, but do not link.
   -e, --execute        Execute program as a script.
   -l, --library LIB    Compile and link with native library LIB.
   -o, --output FILE    Write compiler output to FILE.
   -i, --include DIR    Add the directory DIR to the search path.
   -m, --make           Compile input files and out-of-date dependencies.
   -h, --help           Print this help message.
   -v, --verbose        Print extra information during compilation.
   -g, --generator GEN  Use code generator GEN.
   --build-dir DIR      Output directory for object files.
   --dump-ir            Output the intermediate representation.
   --dump-ast           Output the abstract syntax tree.
   --dump-liveness      Output liveness info when printing the IR.
   --version            Print the compiler version number.

Usage: jogo [OPTIONS] FILE...

   -a, --assembly       Compile, but do not assemble or link.
   -c, --compile        Compile and assemble, but do not link.
   -e, --execute        Execute program as a script.
   -l, --library LIB    Compile and link with native library LIB.
   -o, --output FILE    Write compiler output to FILE.
   -i, --include DIR    Add the directory DIR to the search path.
   -m, --make           Compile input files and out-of-date dependencies.
   -h, --help           Print this help message.
   -v, --verbose        Print extra information during compilation.
   -g, --generator GEN  Use code generator GEN.
   --build-dir DIR      Output directory for object files.
   --dump-ir            Output the intermediate representation.
   --dump-ast           Output the abstract syntax tree.
   --dump-liveness      Output liveness info when printing the IR.
   --version            Print the compiler version number.

